spec/features/requester_edits_ncr_work_order_spec.rb

```diff
@@ -7,10 +7,11 @@
 
   let(:work_order) { create(:ncr_work_order, description: 'test') }
   let(:ncr_proposal) { work_order.proposal }
-  let!(:approver) { create(:user) }
+  let!(:approver) { create(:user, client_slug: 'ncr') }
```

Comment:
not so much about the typing, more about making sure we don't have typos. If we have a typo on the string, it won't throw an error. If we have a typo on a constant, it will throw an `undefined constant` erorr